litellm/utils.py

```diff
@@ -1333,27 +1333,42 @@ def success_handler(
                             end_time=end_time,
                             print_verbose=print_verbose,
                         )
-                    if callback == "llmonitor":
-                        print_verbose("reaches llmonitor for logging!")
+                    if callback == "lunary":
+                        print_verbose("reaches lunary for logging!")
                         model = self.model
+                        kwargs = self.model_call_details
 
-                        input = self.model_call_details.get(
-                            "messages", self.model_call_details.get("input", None)
+                        input = kwargs.get(
+                            "messages", kwargs.get("input", None)
                         )
 
-                        # if contains input, it's 'embedding', otherwise 'llm'
                         type = (
                             "embed"
                             if self.call_type == CallTypes.embedding.value
                             else "llm"
                         )
 
-                        llmonitorLogger.log_event(
+                         # this only logs streaming once, complete_streaming_response exists i.e when stream ends
+                        if self.stream:
+                            print_verbose("reaches lunary for streaming logging!")
+                            print(kwargs)
```

Comment:
clean up the print statements please @vincelwt

Reply:
Ah yes sorry, done! Thanks @krrishdholakia :)